### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
# Description
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

# Submitter checklist
- [ ] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

Note:
1. There are a couple of API changes here (not all have @Deprecated markings)
1. This isn't tested
1. I can split things (especially the API changes) if requested, however, right now splitting them will incur merge bits, which I'd rather avoid.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/TypeUtil.java [hunk 1]
```diff
@@ -86,7 +86,7 @@ public Class onClass(Class c,Void _) {
             return c;
         }
 
-        public Class onParameterizdType(ParameterizedType p,Void _) {
+        public Class onParameterizedType(ParameterizedType p,Void _) {
```
Comment (jsoref) [inline blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/TypeUtil.java]: Probably an API break

File: blueocean-core-js/src/js/fetch.js [hunk 2]
```diff
@@ -272,7 +272,7 @@ export const Fetch = {
      * @param {Object} [options.fetchOptions] - Optional isomorphic-fetch options.
      * @returns JSON body.
      */
-    fetchJSON(url, { onSuccess, onError, fetchOptions, disableCapabilites, disableLoadingIndicator, ignoreRefreshHeader } = {}) {
+    fetchJSON(url, { onSuccess, onError, fetchOptions, disableCapabilities, disableLoadingIndicator, ignoreRefreshHeader } = {}) {
```
Comment (jsoref) [inline blueocean-core-js/src/js/fetch.js]: Probably an API break

File: blueocean-pipeline-editor/src/main/js/PipelineCreateButton.jsx [hunk 3]
```diff
@@ -10,7 +10,7 @@ export default class PipelineCreateButton extends React.Component {
 }
 
 PipelineCreateButton.propTypes = {
-    prganization: React.PropTypes.string,
+    organization: React.PropTypes.string,
```
Comment (jsoref) [inline blueocean-pipeline-editor/src/main/js/PipelineCreateButton.jsx]: I'm hoping this is a bug fix...

File: blueocean-dashboard/src/main/js/util/UpdateGetParam.js [hunk 4]
```diff
@@ -25,3 +25,10 @@ export default function updateGetParamm(paramName, newParamValue, getParams) {
 
     return updatedParamsStr.slice(0, -1);
 }
+
+/*
+ * @deprecated
+ */
+export function updateGetParamm(paramName, newParamValue, getParams) {
```
Comment (jsoref) [inline blueocean-dashboard/src/main/js/util/UpdateGetParam.js]: I hope this works... It's of course a shim for the api change

File: blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubRespositoryContainer.java [hunk 5]
```diff
@@ -1,65 +1,11 @@
...
-import io.jenkins.blueocean.rest.Reachable;
-import io.jenkins.blueocean.rest.hal.Link;
-import io.jenkins.blueocean.rest.impl.pipeline.scm.Scm;
-import io.jenkins.blueocean.rest.impl.pipeline.scm.ScmRepositories;
-import io.jenkins.blueocean.rest.impl.pipeline.scm.ScmRepository;
-import io.jenkins.blueocean.rest.impl.pipeline.scm.ScmRepositoryContainer;
-
-import java.io.IOException;
-import java.net.HttpURLConnection;
+import io.jenkins.blueocean.blueocean_github_pipeline.GithubRepositoryContainer;
```
Comment (jsoref) [inline blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubRespositoryContainer.java]: I hope this shim works for the API change

File: blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractMultiBranchCreateRequest.java [hunk 6]
```diff
@@ -120,16 +120,25 @@ public BluePipeline create(@Nonnull BlueOrganization organization, @Nonnull Reac
...
+        public boolean isJenkinsfileFound() {
             return jenkinsFileFound.get();
         }
+
+        /**
+         * Use {@link #isJenkinsfileFound()} instead.
+         * @deprecated in ?.?
+         */
+        @Deprecated
+        public boolean isJekinsfileFound() {
```
Comment (jsoref) [inline blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractMultiBranchCreateRequest.java]: This is an api change

File: blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/LogResource.java [hunk 7]
```diff
@@ -19,7 +19,7 @@
  * @author Vivek Pandey
  */
 public class LogResource{
-    public static final long DEFAULT_LOG_THREASHOLD = 150;
+    public static final long DEFAULT_LOG_THRESHOLD = 150;
```
Comment (jsoref) [inline blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/LogResource.java]: This is an api change. I hope I don't need to include a thing for it...

File: blueocean-rest/README.md [hunk 8]
```diff
@@ -159,7 +159,7 @@ _errors.field_ - Name of the field in request. Top level field name are represen
 
 _Field Error codes_
 
-| Field Error Code  | Descriptiojn  |
+| Field Error Code  | Description  |
```
Comment (jsoref) [inline blueocean-rest/README.md]: Looks like I'll have to add a trailing space to fix the table width

File: blueocean-rest/README.md [hunk 9]
```diff
@@ -2114,7 +2114,7 @@ curl -XGET -u xxx:yyy http://localhost:8080/jenkins/blue/rest/organizations/jenk
 curl -v -u xxx:yyy http://localhost:8080/jenkins/blue/rest/organizations/jenkins/scm/github/organizations/CloudBees-community/repositories/?credentialId=github&pageSize=10&pageNumber=3
 
 {
-  "_class" : "io.jenkins.blueocean.blueocean_github_pipeline.GithubRespositoryContainer",
+  "_class" : "io.jenkins.blueocean.blueocean_github_pipeline.GithubRepositoryContainer",
```
Comment (jsoref) [inline blueocean-rest/README.md]: This content scares me, as it hints that this class is exposed to the outside world...

File: blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueMultiBranchPipeline.java [hunk 10]
```diff
@@ -21,10 +21,10 @@
 public abstract class BlueMultiBranchPipeline extends BluePipelineFolder{
     public static final String TOTAL_NUMBER_OF_BRANCHES="totalNumberOfBranches";
     public static final String NUMBER_OF_FAILING_BRANCHES="numberOfFailingBranches";
-    public static final String NUMBER_OF_SUCCESSFULT_BRANCHES="numberOfSuccessfulBranches";
+    public static final String NUMBER_OF_SUCCESSFUL_BRANCHES="numberOfSuccessfulBranches";
```
Comment (jsoref) [inline blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BlueMultiBranchPipeline.java]: I hope this is just for localization access and isn't actually public.

File: docu/I18N.md [hunk 11]
```diff
@@ -22,7 +22,7 @@ g co JENKINS-35845
 g nb JENKINS-39225_fr
 ```
 
-Than create a locale version for your translatation. In our case we will create ```fr```
+Then create a locale version for your translation. In our case we will create ```fr```
```
Comment (jsoref) [inline docu/I18N.md]: This file needs a rewrite (or at least a Grammarly pass). It's on my todo list.

File: acceptance-tests/runner/runtime-plugins/pom.xml [hunk 12]
```diff
@@ -3,14 +3,14 @@
...
-  <artifactId>blueocean-acceptance-tests-parrent</artifactId>
+  <artifactId>blueocean-acceptance-tests-parent</artifactId>
   <version>2.0-SNAPSHOT</version>
   <packaging>pom</packaging>
 
   <name>Blue Ocean Parent</name>
 
   <modules>
-    <module>fake-securitly-realm-plugin</module>
+    <module>fake-security-realm-plugin</module>
```
Comment (michaelneale) [inline acceptance-tests/runner/runtime-plugins/pom.xml]: No this won't work as the typo is is actually in the module name as well!

File: acceptance-tests/runner/runtime/pom.xml [hunk 13]
```diff
@@ -8,7 +8,7 @@
     <version>1.32</version>
   </parent>
 
-  <artifactId>bluecoean-acceptance-test</artifactId>
+  <artifactId>blueocean-acceptance-test</artifactId>
```
Comment (michaelneale) [inline acceptance-tests/runner/runtime/pom.xml]: @imeredith heh -

**Comment by michaelneale** [comment]
hey @vivek @imeredith this has a bunch of typos fixed in ATH, and other things that may be worth looking at (I expect the build will fail)

**Comment by jsoref** [comment]
@michaelneale: the tools are in the url in the description. It's run by me manually, and it's fairly easy for me to mess up. You have a working CI, so I'd definitely make sure it's green before expecting anyone to pull it in...

@vivek: certainly. I'm going to start by just getting this set building and dropping stuff that doesn't belong, then i'll reorder and split.

File: blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubRespositoryContainer.java [hunk 14]
```diff
@@ -1,65 +1,11 @@
...
-            final GHRepoEx repository = GithubScm.om.readValue(HttpRequest.getInputStream(connection), GHRepoEx.class);
-            return new GithubRepository(repository, credentials, this);
-        } catch (IOException e) {
-            throw new ServiceException.UnexpectedErrorException(e.getMessage(),e);
-        }
+@Deprecated
+public class GithubRespositoryContainer extends GithubRepositoryContainer {
 
-    }
-}
```
Comment (jglick) [inline blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubRespositoryContainer.java]: missing `}`

File: acceptance-tests/runner/runtime-plugins/pom.xml (hunk 12 above)
Comment (jsoref) [reply acceptance-tests/runner/runtime-plugins/pom.xml]: Fixed

File: acceptance-tests/runner/runtime-plugins/fake-securitly-realm-plugin/src/main/java/org/jenkinsci/plugins/fakerealm/Listerner.java [hunk 15]
```diff
@@ -20,7 +20,7 @@
 import java.util.*;
 
 @Extension
-public class Listerner  extends ComputerListener{
+public class Listener  extends ComputerListener{
```
[the remark this answers is not in the document]
Comment (jsoref) [reply acceptance-tests/runner/runtime-plugins/fake-securitly-realm-plugin/src/main/java/org/jenkinsci/plugins/fakerealm/Listerner.java]: This is done, I'm not sure why github doesn't show this as obsolete.
I'm also not sure why there are two spaces before `extends`. Although in general, I'd rather not make unrelated changes to files.

File: blueocean-commons/src/test/java/io/jenkins/blueocean/commons/stapler/export/ModelTest.java [hunk 16]
```diff
@@ -42,7 +42,7 @@
     ModelBuilder builder = new ModelBuilder();
 
     @Test // JENKINS-26775
-    public void sytheticMethodShouldNotBeExported() {
+    public void syntheticMethodShouldNotBeExported() {
```
[the remark this answers is not in the document]
Comment (jsoref) [reply blueocean-commons/src/test/java/io/jenkins/blueocean/commons/stapler/export/ModelTest.java]: This is dropped, I'm not sure why github doesn't show this as obsolete.

**Comment by jsoref** [comment]
I'm punting on code coverage. I don't understand what jacoco is thinking and do not want to spend more time fighting it.

In general, I tend to split out API changes anyway, and I only offer them as a courtesy.

Fwiw, I plan to split out ATH first, and then doc, and finally whatever remains can live in this PR.

File: blueocean-dashboard/src/main/js/creation/credentials/CreateCredentialDialog.jsx [hunk 17]
```diff
@@ -198,7 +198,7 @@ export class CreateCredentialDialog extends React.Component {
         const disabled = this.state.creationPending;
 
         const buttons = [
-            <button className="button-create-credental" disabled={disabled} onClick={() => this._onCreateClick()}>
```
Comment (jsoref) [inline blueocean-dashboard/src/main/js/creation/credentials/CreateCredentialDialog.jsx]: This change is tied to corresponding changes in ATH.

**Comment by jsoref** [comment]
@vivek: I can't do that completely. Some code changes are tied to changes in the tests...

But I'll try to push those bits to the other side...

**Comment by jsoref** [comment]
Now that this passes tests...
#1670 has most of the acceptance test work (tests passing) -- and those changes are excluded from this PR.
#1672 has most of the comments (tests should pass)
-- I haven't dropped the changes from #1672 from this PR yet, but I can pretty easily.
-- There are some changes for *pipeline* and *jenkins* that could be pushed over to the comments commit, but I think it's easier for a reviewer here to just skip by them, than for me to spend the time moving them over.

**Comment by vivek** [comment]
@jsoref ok to close this?

**Comment by jsoref** [comment]
@vivek : no, i'll update this to exclude the comments bits and then you guys will get to review the fun parts.

**Comment by vivek** [comment]
@jsoref 👍 spell fix in code needs through review, special if it has impact for anyone extending blueocean or consuming its extensions. In fact I am not too keen on code spell fixes (like variable names etc.), but I am going to wait till you have this PR ready for review.

**Comment by kzantow** [comment]
@vivek basically agree about breaking API changes, though, any spelling fixes are very much appreciated. We don't want another `referer` ;)

**Comment by jsoref** [comment]
@viksharma / @micaelgallego / @kzantow : please review...

File: blueocean-core-js/src/js/fetch.js (hunk 2 above)
Comment (kzantow) [reply blueocean-core-js/src/js/fetch.js]: except caught elsewhere, as i've seen

File: blueocean-dashboard/src/main/js/creation/credentials/CreateCredentialDialog.jsx (hunk 17 above)
Comment (kzantow) [reply blueocean-dashboard/src/main/js/creation/credentials/CreateCredentialDialog.jsx]: 👍  was looking for that one

File: blueocean-dashboard/src/main/js/util/UpdateGetParam.js (hunk 4 above)
Comment (kzantow) [reply blueocean-dashboard/src/main/js/util/UpdateGetParam.js]: these classes are not used outside of this project, so _shouldn't_ be a problem

Comment (kzantow) [reply blueocean-dashboard/src/main/js/util/UpdateGetParam.js]: maybe grep through the code for `updateGetParamm` and then bash someone on the head who actually wrote that code ;)

File: blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractMultiBranchCreateRequest.java (hunk 6 above)
Comment (kzantow) [reply blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractMultiBranchCreateRequest.java]: again, only within the project

Comment (kzantow) [reply blueocean-pipeline-scm-api/src/main/java/io/jenkins/blueocean/scm/api/AbstractMultiBranchCreateRequest.java]: ... probably

File: blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/rest/UserImplPermissionTest.java [hunk 18]
```diff
@@ -167,19 +167,19 @@ private void checkPermissions(BlueUserPermission permission, boolean shouldBeAdm
...
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(BluePipeline.STOP_PERMISSION));
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(BluePipeline.CONFIGURE_PERMISSION));
+
+        permissions = permission.getCredentialPermission();
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(CredentialsProvider.CREATE.name.toLowerCase()));
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(CredentialsProvider.VIEW.name.toLowerCase()));
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(CredentialsProvider.DELETE.name.toLowerCase()));
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(CredentialsProvider.UPDATE.name.toLowerCase()));
+        assertEquals("User permission does not match", shouldHaveOtherPermissions, permissions.get(StringUtils.uncapitalize(CredentialsProvider.MANAGE_DOMAINS.name)));
```
Comment (kzantow) [inline blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/rest/UserImplPermissionTest.java]: ... just wow :)

File: js-extensions/spec/ExtensionStore-spec.js [hunk 19]
```diff
@@ -92,7 +92,7 @@ describe("ExtensionStore.js", function () {
             var extensionStore = makeExtensionStore(plugins, false);
 
             try {
-                extenstionStore.getExtensions('ext', function(ext) { });
+                extensionStore.getExtensions('ext', function(ext) { });
```
Comment (kzantow) [inline js-extensions/spec/ExtensionStore-spec.js]: funny, exception thrown... because of the misspelling. *sigh*

Comment (jsoref) [reply js-extensions/spec/ExtensionStore-spec.js]: It's a pretty common problem. I found one of these in a major js testing suite's tests...

It's why this tooling is worth applying...

**Comment by jsoref** [comment]
I'm happy to drop the shims. I only offer them occasionally to projects that seem to care about APIs and compatibility. It's obviously much more work for me to create them, but I certainly an not attached to them. On the other hand, it's much easier for me to create them than to try to prove they aren't necessary. My goal is to provide a digestable and acceptable series of changes.

**Comment by jsoref** [comment]
@kzantow : the misspelled java classes were such a pain, that I decided to leave them for later.

For reference, that's:
* `GithubRespositoryContainer`

And it'd be helpful to know if you'd want a shim for that or not (I currently have the shim, but, I'm of course happy to drop it...).

**Comment by jsoref** [comment]
@micaelgallego / @vivek : can you please return to this PR?

**Comment by vivek** [comment]
@jsoref Sorry for not getting back earlier. Can you rebase and push with latest master?

**Comment by jsoref** [comment]
@vivek: done

**Comment by AnneTheAgile** [comment]
fyi ^^ failure is; https://ci.blueocean.io/blue/organizations/jenkins/blueocean/detail/PR-1664/22/tests
````
New failing - 1
ATH - Jenkins 2.121.1 / restartStageTest – io.blueocean.ath.offline.multibranch.RestartStageTest25s
Error
Error while waiting for something
Stacktrace
io.blueocean.ath.AcceptanceTestException: Error while waiting for something
at io.blueocean.ath.offline.multibranch.RestartStageTest.restartStageTest(RestartStageTest.java:71)
Caused by: org.openqa.selenium.TimeoutException:
Timed out after 20 seconds waiting for io.blueocean.ath.WaitUtil$$Lambda$1/1677568775@4debbf0
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'ef5b7d5839f2', ip: '172.17.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-1048-aws', java.version: '1.8.0_131'
Driver info: driver.version: unknown
at io.blueocean.ath.offline.multibranch.RestartStageTest.restartStageTest(RestartStageTest.java:71)
Caused by: org.openqa.selenium.NoSuchElementException:
no such element: Unable to locate element: {"method":"css selector","selector":".progress-spinner.not_built"}
(Session info: chrome=67.0.3396.99)
(Driver info: chromedriver=2.40.565383 (76257d1ab79276b2d53ee976b2c3e3b9f335cde7),platform=Linux 4.4.0-1048-aws x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 12 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'ef5b7d5839f2', ip: '172.17.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-1048-aws', java.version: '1.8.0_131'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities [{mobileEmulationEnabled=false, hasTouchScreen=false, platform=LINUX, acceptSslCerts=false, acceptInsecureCerts=false, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=, applicationCacheEnabled=false, rotatable=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.40.565383 (76257d1ab79276b2d53ee976b2c3e3b9f335cde7), userDataDir=/tmp/.org.chromium.Chromium.fjhtXq}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, version=67.0.3396.99, browserConnectionEnabled=false, nativeEvents=true, webdriver.remote.sessionid=ba0579418c41ac64efcd50ec246adda9, locationContextEnabled=true, cssSelectorsEnabled=true}]
Session ID: ba0579418c41ac64efcd50ec246adda9
*** Element info: {Using=css selector, value=.progress-spinner.not_built}
at io.blueocean.ath.offline.multibranch.RestartStageTest.restartStageTest(RestartStageTest.java:71)
Standard Output
19:57:09 INFO - Running test: 'restartStageTest'
19:57:09 INFO - SSE Connected ath-5392782398591340605
19:57:09 INFO - Created git repository at /tmp/junit5539895133831228234/junit5611546141855006007
19:57:09 INFO - Committed Jenkinsfile for RestartStageTest
19:57:09 INFO - Created multibranch pipeline: RestartStageTest
19:57:09 INFO - Finished creating RestartStageTest
19:57:09 INFO - Beginning restartStageTest()
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:09 INFO - Build for RestartStageTest entered queue
19:57:10 INFO - Opened RunDetailsPipeline page for RestartStageTest
19:57:10 INFO - Waiting for 'RestartStageTest/master' - queueID:105 to finish
19:57:11 INFO - Build 'RestartStageTest/master' - #1 is finished
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:12 INFO - Build for RestartStageTest entered queue
19:57:13 INFO - Waiting for 'RestartStageTest/master' - queueID:107 to finish
19:57:14 INFO - Build 'RestartStageTest/master' - #2 is finished
19:57:34 INFO - Wrote screenshot to /home/jenkins/slave/workspace/blueocean_PR-1664-ICCG62O2TT4EH44PZ7ZFNPNE5JX3JIOAVX6NH5LTE3ZWXFPTBMLQ/acceptance-tests/target/screenshots/io.blueocean.ath.offline.multibranch.RestartStageTest_restartStageTest.png
19:57:34 INFO - browser console output below:
19:57:34 INFO - 1537387031798 - SEVERE - http://172.17.0.2:50274/blue/rest/organizations/jenkins/pipelines/RestartStageTest/branches/master/runs/1/blueTestSummary/ - Failed to load resource: the server responded with a status of 404 (Not Found)
19:57:34 INFO - 1537387031826 - INFO - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57128:24 "There has been an error while trying to get the TestSummary data."
19:57:34 INFO - 1537387034178 - SEVERE - http://172.17.0.2:50274/blue/rest/organizations/jenkins/pipelines/RestartStageTest/branches/master/runs/1/blueTestSummary/ - Failed to load resource: the server responded with a status of 404 (Not Found)
19:57:34 INFO - 1537387034178 - INFO - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57128:24 "There has been an error while trying to get the TestSummary data."
19:57:34 INFO - 1537387034178 - SEVERE - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean.js 58095:12 "Unhandled Rejection: \"TypeError: Cannot read property 'length' of null\\n at cause (http://172.17.0.2:50274/adjuncts/126ee53a/org/jenkins/ui/jsmodules/blueocean-dashboard/jenkins-js-extension.js:91626:51)\\n at RunDetailsHeader.render (http://172.17.0.2:50274/adjuncts/126ee53a/org/jenkins/ui/jsmodules/blueocean-dashboard/jenkins-js-extension.js:91687:21)\\n at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49621:30)\\n at ReactCompositeComponentWrapper._renderValidatedComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49644:32)\\n at ReactCompositeComponentWrapper._updateRenderedComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49568:36)\\n at ReactCompositeComponentWrapper._performComponentUpdate (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49546:10)\\n at ReactCompositeComponentWrapper.updateComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49467:12)\\n at ReactCompositeComponentWrapper.receiveComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49369:10)\\n at Object.receiveComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:55499:22)\\n at Object.updateChildren (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:48731:25)\\n at ReactDOMComponent._reconcilerUpdateChildren (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:54357:28)\\n at ReactDOMComponent._updateChildren (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:54456:31)\\n at ReactDOMComponent.updateChildren (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:54443:12)\\n at ReactDOMComponent._updateDOMChildren (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:50747:12)\\n at ReactDOMComponent.updateComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:50565:10)\\n at ReactDOMComponent.receiveComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:50527:10)\\n at Object.receiveComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:55499:22)\\n at ReactCompositeComponentWrapper._updateRenderedComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49576:23)\\n at ReactCompositeComponentWrapper._performComponentUpdate (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49546:10)\\n at ReactCompositeComponentWrapper.updateComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49467:12)\\n at ReactCompositeComponentWrapper.receiveComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:49369:10)\\n at Object.receiveComponent (http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/jenkins-design-language.js:55499:22)\""
19:57:34 INFO - 1537387034178 - INFO - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57128:24 "There has been an error while trying to get the TestSummary data."
19:57:34 INFO - 1537387034178 - SEVERE - http://172.17.0.2:50274/blue/rest/search/?q=type:pipeline;organization:jenkins;pipeline:*RestartStageTest*;excludedFromFlattening:jenkins.branch.MultiBranchProject,hudson.matrix.MatrixProject&filter=no-folders&start=0&limit=1 - Failed to load resource: the server responded with a status of 500 (Server Error)
19:57:34 INFO - 1537387034178 - SEVERE - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57967:24 "Error fetching page"
19:57:34 INFO - 1537387034178 - SEVERE - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57967:24 "Error fetching page"
19:57:34 INFO - 1537387034178 - SEVERE - http://172.17.0.2:50274/blue/rest/organizations/jenkins/pipelines/RestartStageTest/branches/master/runs/2/blueTestSummary/ - Failed to load resource: the server responded with a status of 404 (Not Found)
19:57:34 INFO - 1537387034178 - INFO - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57128:24 "There has been an error while trying to get the TestSummary data."
19:57:34 INFO - 1537387034178 - INFO - http://172.17.0.2:50274/adjuncts/126ee53a/io/jenkins/blueocean/blueocean-core-js.js 57128:24 "There has been an error while trying to get the TestSummary data."

````

**Comment by AnneTheAgile** [comment]
list of items before rename; :wow:
ellapsed
isJekinsfileFound
disableCapabilites
jesnkins
parametrised
pipelne
_initalize
updateGetParamm
organiztionPipelinesPager
initilaize
shouldSuceed
isJekinsfileFound
credetialsWrapper
changetSet
scipt
pipline
prganization
disableCapabilites
renderPiplineFlat
stepDetailssContainerStyle
//this ^^ is up through line 152
renderPiplineFlat
foundJekinsFile
isJekinsfileFound
encondedDisplayName
DEFAULT_LOG_THREASHOLD
artfictPath
premissions
tempalte
NUMBER_OF_SUCCESSFULT_BRANCHES
NUMBER_OF_SUCCESSFULT_PULL_REQUESTS
piplelines
jeninks
parametrised
ellapsed
inflectiontOffset
progessBarStory
progessBarStories
extenstionStore
Dfinbugs

**Comment by vivek** [comment]
@imeredith @NicuPascu looks good to you?

**Pull request merged**